From 6ed66ca7532ae83ab3f8b7289f94073738a4497c Mon Sep 17 00:00:00 2001
From: Sam Hartman <hartmans@debian.org>
Date: Mon, 26 Dec 2011 18:05:13 -0500
Subject: =?UTF-8?q?ticket:=20new=0Asubject:=20fix=20ksu=20environment=20va?=
 =?UTF-8?q?riable=20handling?=

Fix error messages from ksu

patch-name: ksu-fix-env-errors
---
 src/clients/ksu/ksu.h | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/clients/ksu/ksu.h b/src/clients/ksu/ksu.h
index f2c0811..cbc4a57 100644
--- a/src/clients/ksu/ksu.h
+++ b/src/clients/ksu/ksu.h
@@ -55,8 +55,12 @@
 
 #define CHUNK 3
 #define CACHE_MODE 0600
-#define MAX_CMD 2048 /* this is temp, should use realloc instead,
-                        as done in most of the code */
+#define MAX_CMD 2048 /* this is temp, should use realloc instead,          
+			as done in most of the code */       
+
+#ifndef MAXPATHLEN
+# define MAXPATHLEN 4096
+#endif		      
 
 
 extern int optind;
